python/test/gherkin_test.py

```diff
@@ -89,3 +91,202 @@ def test_change_the_default_language():
+  parser = Parser()
+  matcher = GherkinInMarkdownTokenMatcher()
+  markdown = """## Feature: DataTables
+
+### Scenario: minimalistic
+
+* Given a simple data table 
+  | foo | bar |
+  | --- | --- |
+  | boz | boo |
+"""
+  ast = parser.parse(TokenScanner(markdown), matcher)
+  expected = {
+        "comments": [
+          {
+            "location": {
+              "column": 3,
+              "line": 7
+            },
+            "text": None
+          }
+        ],
+        "feature": {
+          "children": [
+            {
+              "scenario": {
+                "description": "",
+                "examples": [],
+                "id": "3",
+                "keyword": "Scenario",
+                "location": {
+                  "column": 5,
+                  "line": 3
+                },
+                "name": "minimalistic",
+                "steps": [
+                  {
+                    "dataTable": {
+                      "location": {
+                        "column": 3,
+                        "line": 6
+                      },
+                      "rows": [
+                        {
+                          "cells": [
+                            {
+                              "location": {
+                                "column": 5,
+                                "line": 6
+                              },
+                              "value": "foo"
+                            },
+                            {
+                              "location": {
+                                "column": 11,
+                                "line": 6
+                              },
+                              "value": "bar"
+                            }
+                          ],
+                          "id": "0",
+                          "location": {
+                            "column": 3,
+                            "line": 6
+                          }
+                        },
+                        {
+                          "cells": [
+                            {
+                              "location": {
+                                "column": 5,
+                                "line": 8
+                              },
+                              "value": "boz"
+                            },
+                            {
+                              "location": {
+                                "column": 11,
+                                "line": 8
+                              },
+                              "value": "boo"
+                            }
+                          ],
+                          "id": "1",
+                          "location": {
+                            "column": 3,
+                            "line": 8
+                          }
+                        }
+                      ]
+                    },
+                    "id": "2",
+                    # "docString": None,
```

Comment:
Note this is not included in this implementation - discrepancy with the Javascript implementation